frontend/src/components/UserTables/DispatcherTable.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect, Dispatch, SetStateAction } from 'react';
```

Comment:
Getting build warnings that Dispatch and SetStateAction are never used. Please remove.